docs/src/architecture/accounts.md

```diff
@@ -1,5 +1,8 @@
 # Accounts
-An account is an entity which holds assets and defines rules of how these assets can be transferred. The diagram below illustrates basic components of an account. In Miden every account is a smart contract.
+Miden aims to support expressive smart contracts via a Turing-complete language. Currently there is no better design for smart contracts than account-based state. In Miden, an account is an entity which holds assets and defines rules of how these assets can be transferred. They are basic building blocks representing a user or an autonomous smart contract.
```

Comment:
"Currently there is no better design for smart contracts than account-based state." sounds a bit opinionated.  Can we make it more objective?

Reply:
Very well. This needs to change in a separate PR. This PR is only about notes, it just didn't rebase properly. Will change in a separate PR.

Reply:
Changed it in Accounts PR https://github.com/0xPolygonMiden/miden-base/pull/160